components/python/pyzmq/Makefile

```diff
@@ -10,15 +10,32 @@
 
 #
 # Copyright (c) 2014 Alexander Pyhalov.  All rights reserved.
+# Copyright 2016 Jim Klimov
 #
 
+# See zeromq configure.ac for ABI version mapping to software version
+# NOTE: This requires functionality offered by libzmq-4.1.2 (and hopefully
+# subsequent releases) which we package in a dedicated subtree
+NEEDED_ZEROMQ_VER = 4.1
+
+# The zeromq components define this directory structure
+ZEROMQ_DIR = $(USRDIR)/zeromq-$(NEEDED_ZEROMQ_VER)
+
+# We define these before including any common structure makefiles
+CXXFLAGS   = -I$(ZEROMQ_DIR)/include
+CPPFLAGS   = -I$(ZEROMQ_DIR)/include
+CFLAGS     = -I$(ZEROMQ_DIR)/include
+LDFLAGS.32 = -L$(ZEROMQ_DIR)/lib
+LDFLAGS.64 = -L$(ZEROMQ_DIR)/lib/$(MACH64)
```

Comment:
Sounds complicated... anyhow a bump of PyZMQ requires ZeroMQ-4.1.2 (or newer hopefully too). Why not do both at once and so ensure we have a working level in the gate.

Though indeed the matrix to test all installable libzmq versions vs build and/or installation of all consumer versions (whether requiring a specific ABI or not), sounds like a large undertaking :\